torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -568,6 +568,45 @@ def pointless_view(match: Match, arg, size):
         match.erase_nodes()
 
 
+@register_graph_pattern(
+    CallFunction(
+        aten.view.default,
+        CallFunction(aten.view.default, KeywordArg("arg"), KeywordArg("size1")),
+        KeywordArg("size2"),
+    ),
+    pass_dict=patterns,
+)
+def pointless_view_pair(match: Match, arg, size1, size2):
+    """
+    Remove a pair of views that are pointless.
+    """
+    node = match.output_node()
+    arg_size = list(arg.meta["val"].shape)
+    if arg_size == size2:
```

Comment:
This should be using guard_oblivious or we are going to error on unbacked/add guards